.github/workflows/ci.yaml

```
@@ -43,9 +43,17 @@ jobs:
     needs: [ check-license-header ]
     runs-on: ubuntu-latest
     timeout-minutes: 30
+    strategy:
+      matrix:
+        version: [ 8, 17 ]
```

```suggestion
        version: [ 8, 11, 17 ]
```
I think 11 LTS versions are better.